lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -46,14 +60,12 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      * In Azure B2C, authority is of the form https://{instance}/tfp/{tenant}/{policyName}/
      * Full B2C functionality will be available in this library in future versions.
      *
-     * @param configuration object for the MSAL PublicClientApplication instance
+     * @param configuration {Configuration}
      */
-    constructor(configuration: Configuration) {
-        super(configuration);
-
-        this.activeSilentTokenRequests = new Map();
-        // Register listener functions
-        this.trackPageVisibility = this.trackPageVisibility.bind(this);
+    public static async getInstance(configuration: Configuration): Promise<PublicClientApplication> {
```

Comment:
getInstance is commonly used with getting an instance of a singleton.  This doesn't appear to be a singleton though.  I have a similar method in my PR... you may want to take a look.

Reply:
I am totally fine renaming it to `createPublicClientApplication()`